### PR TITLE
Fix the extend keybinding

### PR DESCRIPTION
**Pull request by DJMcNab**
Make the extend selection keybinding less annoying for users not used to Injelli-J (myself included). Also fixes a minor style issue and runs `npm update`.

### CODE REVIEW

File: editors/code/package.json [hunk 1]
```diff
@@ -28,18 +31,18 @@
         "singleQuote": true
     },
     "dependencies": {
-        "event-stream": "^3.3.4",
+        "event-stream": "^4.0.1",
```
Comment (matklad) [inline editors/code/package.json]: :+1:

File: crates/tools/src/main.rs [hunk 2]
```diff
@@ -122,7 +122,7 @@ fn install_code_extension() -> Result<()> {
         run(r"npm install", "./editors/code")?;
     }
     run(
-        r"node ./node_modules/vsce/out/vsce package",
+        r"npm run package",
```
Comment (matklad) [inline crates/tools/src/main.rs]: Does this actually work? How this calles into `vsce` thigs?

That is, does `cargo install-code` still installes the extension itself?

**Comment by matklad** [comment]
Yeah. If you can write a pre-commit hook as a rust binary in tools and make a corresponding “cargo setup-commit-hook” command, that would be awesome.

**Comment by DJMcNab** [comment]
I've also changed the name of the component we use to `rustfmt` rather than `rustfmt-preview`

bors r+

**Pull request merged**